modules/nodes-descheduler-installing.adoc

```diff
@@ -64,7 +64,21 @@ ifdef::nodes[]
 ====
 Do not enable both `TopologyAndDuplicates` and `SoftTopologyAndDuplicates`. Enabling both results in a conflict.
 ====
-... Optional: Expand the *Profile Customizations* section to set a custom pod lifetime value for the `LifecycleAndUtilization` profile. Valid units are `s`, `m`, or `h`. The default pod lifetime is 24 hours.
+... Optional: Expand the *Profile Customizations* section to set optional configurations for the descheduler.
+**** Set a custom pod lifetime value for the `LifecycleAndUtilization` profile. Use the *podLifetime* field to set a numerical value and a valid unit (`s`, `m`, or `h`). The default pod lifetime is 24 hours.
+
+**** Set a custom priority threshold to consider pods for eviction only if their priority is lower than a specified priority level. Use the *thresholdPriority* field to set a numerical priority threshold or use the *thresholdPriorityClassName* field to specify a certain priority class name.
++
+// TODO: Are there defaults for either of the fields that we should call out here?
++
+[NOTE]
+====
+You cannot specify both *thresholdPriority* and *thresholdPriorityClassName* for the descheduler.
+// TODO: Ask Jan - technically you CAN set both. What happens if you do (and what should we say here).
```

Comment:
@ingvagabund You mentioned that people can only do one or the other, but when I tested it, technically you could set both in the UI, and it was also visible afterward when i viewed the cluster kubedescheduler object:

```
...
    profileCustomizations:
      devLowNodeUtilizationThresholds: Medium
      namespaces: {}
      thresholdPriority: 1
      thresholdPriorityClassName: my-class-name
...
```

A few questions:

1. Should that result in an error if both are set?
2. If not, what happens in the backend if both are set? (Which is actually taken/used?)

Reply:
That's up to each strategy to decide if setting both values is valid or not. E.g. https://github.com/kubernetes-sigs/descheduler/blob/master/pkg/descheduler/strategies/duplicates.go#L47-L49. In all the strategies we always consider this case to be invalid.

Reply:
Thanks, I'll change this to "Do not" instead of "You cannot"